lib/SPIRV/libSPIRV/SPIRVOpCode.h

```diff
@@ -135,6 +135,12 @@ inline bool isGroupOpCode(Op OpCode) {
   return OpGroupAll <= OC && OC <= OpGroupSMax;
 }
 
+inline bool isMediaBlockINTELOpcode(Op OpCode) {
```

Comment:
```suggestion
inline bool isMediaBlockINTELOpcode(Op OC) {
```

Reply:
I've left "OpCode" parameter name (to keep consistent with entire file naming convention), but removed assigning it to the local variable OC.